examples/futil/pass-in-register.futil

```diff
@@ -7,23 +7,28 @@ component times_10_and_add_1(
 ) -> (reg_in: 32, reg_write_en: 1) {
```

Comment:
So, this is a good example for why we should probably keep the unsynthesizable primitives around but name them differently? For example, have a primitive library can `unsynthesizable.futil` which has the current `std_mult` primitive and uses the prefix `std_unsyn_` for everything (renaming `std_mult` to `std_unsyn_mult`).

Additionally, we'll have to update the text in the tutorial here and explain that `std_unsyn_` means that this design cannot be synthesized yet.

Reply:
As far as I can tell, no documentation exists with this program. It looks like ANCHORS were placed preemptively.